src/components/TodoContent.tsx

```diff
@@ -0,0 +1,244 @@
+import React, { useEffect, useMemo, useRef, useState } from 'react';
+import { Todo } from '../types/Todo';
+import { TodoList } from './TodoList';
+import {
+  deleteSelectedTodo,
+  getTodos,
+  postTodo,
+  updateTodo,
+  USER_ID,
+} from '../api/todos';
+import { Errors } from '../types/Errors';
+import classNames from 'classnames';
+import { FilterBy } from '../types/FilterBy';
+import { Header } from './Header';
+import { Footer } from './Footer';
+
+export const TodoContent: React.FC = () => {
+  const [todoValue, setTodoValue] = useState<string>('');
+  const [todosFromServer, setTodosFromServer] = useState<Todo[]>([]);
+  const [errorMessage, setErrorMessage] = useState<string>('');
+  const [loadingTodo, setLoadingTodo] = useState<Todo | null>(null);
+  const [completedTodos, setCompletedTodos] = useState<Todo[]>([]);
```

Comment:
you don't need state for this

Reply:
How should I process it better?

Reply:
Why don't I need a state for todosFromServer